src/messageHandlers/index.js

```
@@ -0,0 +1,28 @@
+const handleUserMessage = require("./handleUserMessage");
+const handleGroupMessage = require("./handleGroupMessage");
+
+module.exports = async function handleAllMessages(message) {
+  if (message.ugClass === "user") {
+    // ugClass=user means that a user is created or updated in UG
+    const user = await handleUserMessage(message);
+
+    return {
+      type: "user",
+      user,
+    };
+  }
+
+  if (message.ugClass === "group") {
+    // ugClass=group means that people are added to or removed from an UG group
+    // depending on the group we do different things
+
+    const group = await handleGroupMessage(message);
+
+    return {
+      type: "group",
+      group,
+    };
+  }
+
+  return { type: "other" };
```

I notice that this `type:"other"` is never used anywhere. I think it is a good thing to return it just like you to just to make it more readable, but I see a problem with this code: we will never notice when a message is handled as `other`. Which means that any bugs within this function could cause us to miss every message without noticing.
What does `other` mean? Is `type:other` expected or an exception? Should it be logged as a warning or even an error, or perhaps throw an error instead of returning?

---

That's a good question: what does "other" mean? I don't have the answer for that now. We should discuss about it